main.py

```diff
@@ -633,6 +639,43 @@ def pick_random_emoji() -> str:
     return decoded_random_emoji
 
 
+async def command_form(message: Message):
+    # Escape strings so they can be assigned as literals within appscript
+    def escape_appscript(text: str):
+        return text.replace("\\", "\\\\").replace('"', '\\"')
+
+    # Constants in generated code, Mmke sure these strings are properly escaped
+    default_title = "Busty's Voting"
+    low_string = "OK"
+    high_string = "Masterpiece"
+    low_score = 0
+    high_score = 7
+
+    appscript = "function r(){"
+    # Setup and grab form
+    appscript += f'var f=FormApp.getActiveForm().setTitle("{default_title}");'
+    # Clear existing data on form
+    appscript += "f.getItems().forEach(i=>f.deleteItem(i));"
+    # Add new data to form TODO: properly escape double quotes and backslashes, etc
```

Comment:
What bugs does the current behaviour have?

Reply:
None (that I know of), I did the TODO and forgot to remove it